src/types/helpers.ts

```diff
@@ -1,9 +1,7 @@
 /**
  * Extend this interface to build a type that is treated as an open set of properties, where each key is a string.
  */
-export interface StringIndexed {
-  [key: string]: any;
-}
+export type StringIndexed = Record<string, any>;
```

Comment:
👍

Reply:
👍 Yeah the pr was looking too big already so I opted to drop it here.